Stratis.Bitcoin/Features/MemoryPool/MempoolBehavior.cs

```diff
@@ -13,30 +13,86 @@
 
 namespace Stratis.Bitcoin.Features.MemoryPool
 {
+    /// <summary>
+    /// Node behavior for memory pool.
+    /// </summary>
     public class MempoolBehavior : NodeBehavior
     {
-        // Average delay between trickled inventory transmissions in seconds.
-        //  Blocks and whitelisted receivers bypass this, outbound peers get half this delay. 
+        #region Fields
+
+        /// <summary>
+        /// Average delay between trickled inventory transmissions in seconds.
+        /// Blocks and whitelisted receivers bypass this, outbound peers get half this delay. 
+        /// </summary>
         const int INVENTORY_BROADCAST_INTERVAL = 5;
-        // Maximum number of inventory items to send per transmission.
-        //  Limits the impact of low-fee transaction floods. 
+
+        /// <summary>
+        /// Maximum number of inventory items to send per transmission.
+        /// Limits the impact of low-fee transaction floods.
+        /// </summary>
         const int INVENTORY_BROADCAST_MAX = 7*INVENTORY_BROADCAST_INTERVAL;
 
+        /// <summary>
+        /// Memory pool validator injected dependency.
+        /// </summary>
         private readonly IMempoolValidator validator;
+
+        /// <summary>
+        /// Memory pool manager injected dependency.
+        /// </summary>
         private readonly MempoolManager manager;
```

Comment:
I agree injected dependency is not so helpful, and hopefully everything will be injected. 
More explanation  is probably better.
Example
>The mempool manager contains high level methods that can be used from outside of the mempool.